### PR TITLE
Made a few changes

### PR DESCRIPTION
**Pull request by raceking37**
Hi, I made the following changes:

- Added more bit fields
- removed a couple of semicolons in device.py
- added isLightAvailable() method
- Used the isLightAvailable() method in the test_ambient.py example

### CODE REVIEW

File: apds9960/device.py [hunk 1]
```diff
@@ -23,23 +23,23 @@ def __init__(self, bus, address=APDS9960_I2C_ADDR, valid_id=APDS9960_DEV_ID):
         # instance variables for gesture detection
         self.gesture_ud_delta_ = 0
         self.gesture_lr_delta_ = 0
-
+
```
Comment (liske) [inline apds9960/device.py]: Could you please remove those trailing whitespaces on empty lines? I'm fine with the other changes :-)

**Review by liske — CHANGES_REQUESTED** [review]

File: apds9960/device.py (hunk 1 above)
Comment (raceking37) [reply apds9960/device.py]: They should be gone now 👍

**Review by liske — APPROVED** [review]

**Comment by liske** [comment]
Thanks for your contribution! :smiley:

**Comment by tbuss** [comment]
Hi @liske,
I tried today your lib and installed it from pypi. After some test I figured out, that the version there is missing isLightAvailable(), so it wasn't update there I guess. 
Would you mind to update the module there? Thanks a lot

**Comment by Joseph0M** [comment]
> After some test I figured out, that the version there is missing isLightAvailable()

Same here, I thought my version was corrupted or something, as I couldn't find the function at runtime. This needs to be fixed, especially as the function is included in the Examples!

**Pull request merged**